.github/workflows/win.yml

```diff
@@ -0,0 +1,79 @@
+name: Windows Tests
+
+on:
+  push:
+    branches:
+      - master
+      - maintenance/**
+  pull_request:
+    branches:
+      - master
+      - maintenance/**
+
+jobs:
+  test_windows:
+    name: Python 3.9
+    runs-on: windows-2019
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: clang Version Upgrade
+      run: |
+        choco install -y llvm
+        refreshenv
+        choco install -y mingw --force --version=6.4.0
+        refreshenv
+      shell: pwsh
+
+    - name: Install Python 3.9
+      run: |
+        choco install -y python
+      shell: pwsh
+
+    - name: Install Visual Studio 2019 Build Tools
+      run: |
+        choco install -y visualstudio2019buildtools
+      shell: pwsh
+
+    - name: Install Visual Studio 2019 Workload VC Tools
+      run: |
+        choco install -y visualstudio2019-workload-vctools
+      shell: pwsh
+
+    - name: Install OpenBLAS
+      run: |
+        python -m venv scipy-dev
+        scipy-dev\Scripts\activate
+        python -m pip install --upgrade pip "setuptools<50.0" wheel
+        set NPY_USE_BLAS_ILP64=1
+        $openblas = python tools/openblas_support.py
+        cp $openblas scipy-dev\Lib\openblas64_.a
+        set NPY_USE_BLAS_ILP64=0
+        $openblas = python tools/openblas_support.py
+        cp $openblas scipy-dev\Lib\openblas.a
+        echo "Installed OpenBLAS..."
+      shell: pwsh
+
+    - name: Build SciPy
+      run: |
+        scipy-dev\Scripts\activate
+        python -m pip install cython==0.29.18 matplotlib mpmath numpy==1.19.5 Pillow pybind11 pythran pytest==5.4.3 pytest-cov pytest-env pytest-timeout pytest-xdist==1.34.0
+        set NPY_USE_BLAS_ILP64=1
+        set SCIPY_USE_PYTHRAN=0
```

Comment:
To me this looks like it says not to use Pythran, but Pythran is obviously being used according to the CI logs.  What happens if you set this `SCIPY_USE_PYTHRAN=1`?

Reply:
>  What happens if you set this `SCIPY_USE_PYTHRAN=1`?

I think it gives me the same error. IIRC, I added this as an attempt to get rid of the error.

Reply:
Okay, I'll try to see if I can reproduce this locally by following the pipeline exactly

Reply:
Thanks!

Reply:
I am so far unable to reproduce this locally, but I'll have another go at it later this week

Reply:
FYI, tried again last night in a fresh VM, but running into trouble getting OpenBLAS recognized

Reply:
> FYI, tried again last night in a fresh VM, but running into trouble getting OpenBLAS recognized

Thanks! I think I can propose on SciPy master by next week and we can see if it passes there. Mostly because I think the build failure is unrelated. Until then, I guess, we don't need to worry about this

Reply:
Yep, this shouldn't hold up this PR and no need to work on this further, I was interested in digging in to see if we could resolve the existing pythran issue open in SciPy but might be more trouble than it's worth

Reply:
in pwsh environment set won't cut it. You need `$env:NPY_USE_BLAS_ILP64=1` and so on just like above

Reply:
> in pwsh environment set won't cut it. You need `$env:NPY_USE_BLAS_ILP64=1` and so on just like above

Thanks for letting me know! I didn't touch it mostly because I was apprehensive about unfathmable test failures.